packages/calcite-components/src/components/tip/tip.scss

```diff
@@ -1,23 +1,38 @@
+@import "../../assets/styles/header";
+
+/**
+* CSS Custom Properties  
+*
+* These properties can be overridden using the component's tag as selector.
+*
+* @prop --calcite-tip-border-color: The border color of the component.
+* @prop --calcite-tip-background-color: The background color of the component.
+
+*/
+
 :host {
-  @apply border-color-2
-    bg-foreground-1
-    text-color-2
+  @apply text-color-2
     text-n1h
     relative
-    m-4
     box-border
     flex
     flex-row
     border
-    border-solid;
+    border-solid
+    m-4;
+
+  --calcite-tip-border-color: var(--calcite-color-border-2);
+  --calcite-tip-background-color: var(--calcite-color-foreground-1);
 
   * {
     @apply box-border;
   }
 }
 
 .container {
-  @apply w-full p-4;
+  @apply p-4;
```

Comment:
shouldn't these `p-` and `m-` values be converted to use tokens?

Reply:
leave these as is for now.